### DISCUSSION TITLE
Evdevhook2 in Batocera for 2 controllers

### DISCUSSION THREAD

**Discussion by edbracam** [v1993/evdevhook2#15] [Q&A]
Hi, I did not catch how to manage 2 different controllers at the same time. Evdevhook2 was installed via Batocera-Extra project in the Batocera system. there are the following files:
- /usr/bin/evdevhook2
- /var/run/evdevhook2.ini
I still not manage to know how batocera starts this service. Thank you.

**Comment by v1993** [comment]
I do not know how batocera manages it either. Your question would me more aptly directed to Batocera-Extra.
